src/bacnet/basic/object/sc_netport.c

```diff
@@ -724,6 +801,17 @@ bool Network_Port_SC_Direct_Connect_Binding_Set(uint32_t object_instance,
     snprintf(params->SC_Direct_Connect_Binding,
         sizeof(params->SC_Direct_Connect_Binding), "%s", str);
 
+    return true;
```

Comment:
@Michail-Antropov This difference shows that Network_Port_Changes_Pending_Set() was dropped from the definition of Network_Port_SC_Primary_Hub_URI_Set(). Is that intentional?